samples/web/blazor-server-application/Core_8/WebApp/Pages/Index.razor

```diff
@@ -0,0 +1,52 @@
+﻿@page "/"
+@using Microsoft.Extensions.Logging
+@inject ILogger<Index> Logger;
+@inject IMessageSession messageSession;
+
+<PageTitle>NServiceBus Blazor server</PageTitle>
+
+<div>
+    Enter a number below and click "Go".<br />
+    If the number is even, the result will be "Fail", otherwise it will be "None".
+    <br /><br />
+    <input type="text" @bind="userInput" />
+    <input type="submit" value="Go" @onclick="HandleValidSubmit" />
+    <br />
+    @ResponseText
+    <br />
+</div>
+
+@code {
+    private string? userInput;
+    private string? ResponseText;
+
+    protected async Task HandleValidSubmit()
+    {
+        Logger.LogInformation("HandleValidSubmit called");
+
+        // Process the valid form
+        if (string.IsNullOrWhiteSpace(userInput))
+        {
+            await Task.CompletedTask;
+        }
+
+        #region ActionHandling
+
+        if (!int.TryParse(userInput, out var number))
+        {
+            await Task.CompletedTask;
```

Comment:
If it's not an int, shouldn't this be a return? 

```suggestion
            return;
```

Reply:
You're correct. This should just be a return.